### ISSUE TITLE
Invalid argument: Incompatible shapes: [128,84] vs. [32,84]

### ISSUE THREAD

**Issue by zhl95** [choasUp/SIN#12]
tracing error： 

/tools/../lib/fast_rcnn/train.py", line 113, in _modified_smooth_l1
    inside_mul = tf.multiply(bbox_inside_weights, tf.subtract(bbox_pred, bbox_targets))

**Comment by YANYANYEAH** [comment]
Hi, I have the same problem, have you solved it?

**Comment by zhl95** [comment]
No. I also found other errors in the code. I implemented this SIN in Pytorch but cannot reproduce the claimed mAP.

**Comment by monjurulkarim** [comment]
@zhl95 can you plz share your pytorch implementation?

**Comment by SayantiBardhan** [comment]
@zhl95 Can you please share your pytorch implementation

**Comment by SayantiBardhan** [comment]
> No. I also found other errors in the code. I implemented this SIN in Pytorch but cannot reproduce the claimed mAP.

Can you please share your pytorch implementation

**Comment by veracheug** [comment]
excuse me,do you have  implemented the SIN model,I want to exchange some questions with you，Looking forward to your answer， thank you！@zhl95 @SayantiBardhan @monjurulkarim
